packages/fx-core/src/plugins/resource/frontend/env.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+"use strict";
+
+import * as dotenv from "dotenv";
+import fs from "fs-extra";
+import * as path from "path";
+import * as os from "os";
+import { TelemetryHelper } from "./utils/telemetry-helper";
+import { TelemetryEvent } from "./constants";
+
+export interface RemoteEnvs {
+  teamsfxRemoteEnvs: { [key: string]: string };
+  customizedRemoteEnvs: { [name: string]: string };
+}
+
+export const envFileName = (envName: string): string => `env.teamsfx.${envName}`;
+export const envFilePath = (envName: string, folder: string): string =>
```

Comment:
Nit: Would it be better to use path type instead of string for the `folder` parameter?